e2e/__tests__/globalSetup.test.ts

```diff
@@ -109,7 +109,7 @@ test('should not call a globalSetup of a project if there are no tests to run fr
 
   const result = runWithJson(e2eDir, [
     `--config=${configPath}`,
-    '--testPathPatterns=project-1',
+    '--testPathPatterns=setup1',
```

Comment:
right, since this is relative to the project config now, and not the global config. I'm afraid this will be quite breaking as well (when using jets projects just to have a single `jest` command to run from root rather than as a way to isolate test suites), but let's try! 🙂